Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 plugin "bundler-inject", "~> 1.1"
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
 
+# Parser for Clowder config in ENV['ACG_CONFIG'] path
+gem 'app-common-ruby', :git => 'https://github.com/slemrmartin/app-common-ruby', :branch => 'master', :require => false
```

Comment:
Not a blocker, but we can probably change this to the insights repo right?

Reply:
@lindgrenj6 we want to change it to rubygems, but there is a discussion about the name of gem at this moment